packages/gafl-webapp-service/src/pages/contact/address/lookup/address-lookup.njk

```diff
@@ -7,7 +7,13 @@
 
 {% set title = mssgs.address_lookup_title_you if data.isLicenceForYou else mssgs.address_lookup_title_other %}
 
-{% block pageTitle %}{{ title }}{{ mssgs.gov_uk }}{% endblock %}
+{% block pageTitle %}
+  {% if error %}
+    {{ mssgs.server_error_title_prefix }}{{ title }}{{ mssgs.header_service_name_title }}{{ mssgs.gov_uk }}
+  {% else %}
+    {{ title }}{{ mssgs.header_service_name_title }}{{ mssgs.gov_uk }}
+  {% endif %}
+{% endblock %}
```

Comment:
This would be more concise if you did:
```

 {% block pageTitle %}
   {% if error %}
     {{ mssgs.server_error_title_prefix }}
   {% endif %}
   {{ title }}{{ mssgs.header_service_name_title }}{{ mssgs.gov_uk }}
 {% endblock %}
```
(this applies to all subsequent blocks, too)